man/build.q.set.Rd

```diff
@@ -11,31 +11,34 @@
 
 \arguments{
   \item{q.concourse}{
-  	A matrix with meaningful short handles as rownames, languages as columns, and full items in cells as produced by \code{\link{import.q.concourse}}.
+  	A matrix with (meaningful, short) handles as rownames, languages (if applicable) as columns, and full items in cells as produced by \code{\link{import.q.concourse}}.
   }
   \item{q.sample}{
-  	A character vector of sampled meaningful short handles, such as \code{c("q-is-great","q-is-obscure")}.
+  	A character vector of sampled (meaningful, short) handles, such as \code{c("q-is-great","q-is-obscure")}.
```

Comment:
such as \code{c("q-is-great","q-is-obscure")}

This should appear the first time handles are mentioned (i.e. in q.concourse)